src/Game/ReplayServer.java

```diff
@@ -233,90 +237,110 @@ public void run() {
       Logger.Error("ReplayServer: Failed to serve replay");
     }
   }
-
-  public void replayOutput(ReplayPacket packet) {
-    // DROP_ITEM
-    if (packet.opcode == 246) {
-      int item = -1;
-      try {
-        item = packet.data[0] << 8 | packet.data[1];
-      } catch (Exception e) {
-      }
-      if (item < 0) return;
-      Client.displayMessage(
-          "Dropping " + Item.item_name[Client.inventory_items[item]], Client.CHAT_NONE);
-    }
-    // SELECT_DIALOGUE_OPTION
-    else if (packet.opcode == 116) {
-      int chosen = -1;
-      try {
-        chosen = packet.data[0];
-      } catch (Exception e) {
-      }
-      String[] menuOptions =
-          lastMenu != null && lastMenu.get().size() > 0
-              ? lastMenu.get().toArray(new String[0])
-              : Client.menuOptions;
-      if (chosen < 0) {
-        return;
-      } else if (menuOptions == null || menuOptions[chosen] == null) {
-        lastErrorChosenOpt = chosen;
-        lastErrorChosenOptStamp = packet.timestamp;
-        return;
-      }
-      Client.printSelectedOption(menuOptions, chosen);
-      lastMenu.set(new ArrayList<String>());
-    }
-  }
-
-  public void readInput(ReplayPacket packet) {
-    // SHOW_DIALOGUE_MENU
-    if (packet.opcode == 245) {
-      ArrayList<String> menu = new ArrayList<String>();
-      int numOpts = 0;
-      boolean read = true;
-      byte[] option;
-      try {
-        numOpts = packet.data[0];
-        if (numOpts > 0) {
-          read = true;
-          int start = 1;
-          int end = 1;
-          int cur;
-          while (read) {
-            for (cur = start + 1; cur < packet.data.length; cur++) {
-              if (packet.data[cur] == 0) {
-                end = cur;
-                break;
-              }
-            }
-            // inclusive from, exclusive to
-            option = Arrays.copyOfRange(packet.data, start + 1, end);
-            menu.add(new String(option));
-            start = ++end;
-            if (cur >= packet.data.length || menu.size() == numOpts) {
-              read = false;
-            }
-          }
-        }
-        lastMenu.set(menu);
-      } catch (Exception e) {
-      }
-      if (menu.size() == 0) {
-        return;
-      }
-      Client.printReceivedOptions(menu.toArray(new String[0]), menu.size());
-      // replay back chosen option (if couldnt be played earlier bc some bad sync)
-      if (lastMenu.get().size() > 0
-          && Math.abs(packet.timestamp - lastErrorChosenOptStamp) < 100
-          && lastErrorChosenOpt > 0
-          && lastErrorChosenOpt < lastMenu.get().size()) {
-        Client.printSelectedOption(lastMenu.get().toArray(new String[0]), lastErrorChosenOpt);
-        lastErrorChosenOpt = -1;
-        lastErrorChosenOptStamp = -1;
-      }
-    }
-  }
+	
+	public void replayOutput(ReplayPacket packet) {
```

Comment:
Could you run `ant format-source` so that I can see better what changed in this block? Too many indent changes

Reply:
Ran the format may need to reupdate on comments